test/known_issues/test-cluster-import-scheduling-policy.mjs

```diff
@@ -0,0 +1,12 @@
+// Refs: https://github.com/nodejs/node/issues/49240
+// When importing cluster in ESM, cluster.schedulingPolicy cannot be set;
+// and the env variable doesn't work since imports are hoisted to the top.
+import '../common/index.mjs';
+import assert from 'node:assert';
+import * as cluster from 'cluster';
+
+
+assert.strictEqual(cluster.schedulingPolicy, cluster.SCHED_RR);
+cluster.setupPrimary({ schedulingPolicy: cluster.SCHED_NONE });
+const settings = cluster.getSettings();
```

Comment:
I may be missing some context, but is `getSettings()` actually a thing? Should this be `cluster.settings` instead?